pages/common/awk.md

```diff
@@ -1,36 +1,33 @@
 # awk
 
 > A versatile programming language for working on files.
+> See also: `sed`, `ed`.
 > More information: <https://manned.org/awk.1>.
 
-- Execute an expression:
+- Execute commands:
 
-`awk "{{expression}}"`
+`awk '{{{ print gensub("apple", "mango", "g") }}}'`
 
 - Execute a script:
 
-`awk --file {{path/to/file}}`
+`awk --file {{path/to/script.awk}}`
 
 - Check a script for syntax errors:
 
-`awk --lint --file {{path/to/file}}`
+`awk --lint --file {{path/to/script.awk}}`
 
-- Execute an expression with the specified field separator:
+- Execute commands with the specified field separator:
 
-`awk --field-separator="{{,}}" "{{expression}}"`
+`awk --field-separator="{{,}}" '{{{ print $1 }}}'`
 
-- Execute an expression with the specified variables:
+- Replace a string with the specified replacement for all lines:
 
-`awk --assign "{{variable}}={{value}}" "{{expression}}"`
+`awk '{ print gensub("{{regular_expression}}", "{{replacement}}", "g") }'`
 
 - Print the specified field of each line:
 
-`awk "{ print \$1 }"`
+`awk '{ print ${{1}} }'`
 
 - Print the specified field range of each line:
```

Comment:
I would say that the `Execute an expression with the specified variables` example would be more useful than this one.